PAPER/FIT_Paper_Ex1/Part_3.tex

```diff
@@ -69,58 +69,59 @@ \subsection{Exercise specifications}
-plutonium vector produced from a small burn-up discharge CANDU is characterized
-by a high fissile fraction~\cite{Guillemin_2010}. The multi-recycling of the
-plutonium into PWR may produce a high even isotopes fraction in the plutonium
-vector~\cite{Courtin_2016}. Very specific plutonium isotopic fraction can be
-seen from scenarios involving SFR reactors. Also, industrial parameters, such
-has spent fuel cooling time, may have a strong impact on $^{241}$Pu fraction.
-For those reasons, the plutonium isotopic range has been chosen very large to be
-representative of a wide range of applications although being realistic enough.
-The Table~\ref{Tab:PuVector} present minimum and maximum isotopic fraction of
-plutonium isotopes used in the framework of this work.
+The simulation last a the reactor cycle. 
+%From the
+%relation between the fuel cycle time $\Delta t$, the reactor thermal power
+%$P_{th}$, the heavy nuclide mass $M$ and the reactor burn-up $BU$, we have : 
+%
+%\begin{equation}
+%    \Delta t = \frac{BU \; M}{P_{th}}
+%\end{equation}
+At t = 0, the fabrication plant produces the fresh fuel according to reactor
+requirements. To avoid radioactive decay, the fabrication process and the
+reactor fuel loading are simultaneous and instantaneous. The MOX-fuel stays in
+the reactor for a complete cycle (it reaches it designed \gls{BU} and is then
+send to the back-end storage.
+
+
+As it is expected that, for a set reactor, the MOX-fuel plutonium content should depend on the
+plutonium isotopic composition, a large rage of plutonium isotopic composition
+have been considered, allowing to highlight the different between \gls{FLM} and
+\gls{FF} modeling.
+%Concerning the plutonium vector used to build the \gls{MOX} fuel, a large range has
+%been imposed for plutonium isotopes. Indeed, the impact of a \gls{FLM} in
+%relation to \gls{FF} approach on a fuel cycle calculation will increase with the
+%plutonium fraction deviation from a reference case. To take an example, let
+%consider a small fissile content plutonium. A \gls{FF} approach would for
+%instance load the fuel at 6\% of plutonium in the fresh fuel since a \gls{FLM}
+%approach would give 12\%. On the other hand, if the plutonium vector used to
+%build the fuel is very close from the reference plutonium used to design the
+%\gls{FF} approach, \gls{FF} and \gls{FLM} would produce similar results.
+%Perfectly well tuned scenario for which plutonium stocks are well known and
+%approximately constant with time can be properly solved with a \gls{FF}
+%approach. 
+
+More over the defined isotopic space as been designed to cover a wide range of
+fuel history: a CANDU fuel with low discharge burn-up (high fissile
+fraction~\cite{Guillemin_2010}), \gls{PWR} multi-recycled MOX fuel (high even
+isotopes fraction in the plutonium~\cite{Courtin_2016}), long cooling time (low
+$^{241}$Pu fraction). The Table~\ref{tab:PuVector} presents plutonium isotopic
+space covered in this work.
```

Comment:
Very goude. But I think we miss the idea of : 

"if the plutonium vector used to
	%build the fuel is very close from the reference plutonium used to design the
	%\gls{FF} approach, \gls{FF} and \gls{FLM} would produce similar results."

I mean we need to say that for standard studies, such as PWR-UOx + 10% PWR-MOx, required Pu vector would be more or less constant and the wide Pu range we used may not be representative.